app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
```

Comment:
Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.